k2/python/k2/fsa.py

```diff
@@ -26,6 +26,57 @@ class Fsa(object):
     a vector of FSAs it is a tuple with three
     elements ``(num_fsas, None, None)``.  (Caution: it's possible
     for a vector of FSAs to have zero or one elements).
+
+    An instance of FSA has the following attributes:
+
+    - ``arcs``: You will NOT use it directly in Python. It is an instance
+                of ``_k2.RaggedArc`` with only one method ``values()`` which
+                returns a 2-D `torch.Tensor`` of dtype ``torch.int32`` with 4
+                columns. Its number of rows indicates the number of arcs in the
+                FSA. The first column represents the source states, second
+                column the destination states, third column the labels and the
```

Comment:
as we declared here, `third column the labels`, instead of `third column the symbols`